server/controllers/LibraryItemController.js

```diff
@@ -157,33 +157,49 @@ class LibraryItemController {
    * @param {Response} res
    */
   async download(req, res) {
```

Comment:
Using this endpoint isn't going to work because all of the endpoints in `/api/` are authenticated. The reason it would have worked in your testing is because you're still authenticated via the session. You can test it by using the share link in incognito.

The download functionality should be in the `ShareController` which gets served from the unauthenticated `PublicRouter`. Most of those functions are authenticating using the share_session_id cookie.

Reply:
Ah, got it. I didn't test in an incognito and didn't realize all endpoints in `/api/` were authenticated, but that makes sense! I'll move the download functionality to the ShareController.